frontend/src/pages/ArtifactDetails.test.tsx

```diff
@@ -1,42 +1,41 @@
 import * as React from 'react';
-import ArtifactDetails from './ArtifactDetails';
-import {PageProps} from './Page';
-import {shallow, ShallowWrapper, ReactWrapper} from 'enzyme';
-import {Api} from '../lib/Api';
-import * as TestUtils from '../TestUtils'
-import {RouteParams} from '../components/Router';
-import {ApiGetArtifactResponse} from '../apis/service';
+import ArtifactDetails, { ArtifactDetailsTab } from './ArtifactDetails';
+import { PageProps } from './Page';
```

Comment:
Let's try to stick to the style guide already being used by the codebase?

Reply:
Reverted these changes in the latest version. I added Prettier locally but it wasn't following the existing style. I can add a config that uses the current styles in a separate PR.